src/control/SConscript

```diff
@@ -256,27 +266,15 @@ def scons():
     ldopts = cgolibdirs +                                                     \
              " -lspdk_env_dpdk -lspdk_nvme -lspdk_vmd -lrte_mempool" +        \
              " -lrte_mempool_ring -lrte_bus_pci -lnvme_control -lnuma -ldl"
-    senv.AppendENVPath("CGO_LDFLAGS", ldopts, sep=" ")
+    aenv.AppendENVPath("CGO_LDFLAGS", ldopts, sep=" ")
 
-    senv.AppendENVPath("CGO_CFLAGS",
+    aenv.AppendENVPath("CGO_CFLAGS",
                        senv.subst("-I$SPDK_PREFIX/include "
                                   "-I$OFI_PREFIX/include"),
                        sep=" ")
 
-    # Copy setup_spdk.sh script to be executed at daos_server runtime.
-    senv.Install(join('$PREFIX', 'share/daos/control'),
-                 join(gosrc, 'server/init/setup_spdk.sh'))
-
-    serverbin = install_go_bin(senv, gosrc, [denv.nvmecontrol], "daos_server",
-                               "daos_server")
-
-    #Admin binary is expected to be relocated so origin based paths are useless
-    aenv = senv.Clone()
-    aenv["ENV"]["CGO_LDFLAGS"] = old_cgold
-    aenv.Replace(RPATH=[])
     # Sets CGO_LDFLAGS for rpath
     daos_build.add_rpaths(aenv, None, True, True)
```

Comment:
This line should go before line 260.  Otherwise, daos_admin rpath will not be set correctly and daos_admin rpath is needed for developer build